File: Greetings.py
class Greetings:
    def __init__(self, name):
        self.name = name

    def display(self):
        print(f'Hello there {self.name}.')


if __name__ == '__main__':
    i = input('Enter your name: ')
    g = Greetings(i)
    g.display()

File: Greetings.java
import java.util.Scanner;
public class Greetings
{
	public static void main(String[] args) 
	{
		Scanner name = new Scanner(System.in);
                                        System.out.print("Enter your name: ");
		String i = name.nextLine();
                                        System.out.println("Hello there " + i );
	}

}
